### ISSUE TITLE
feature request: trigger by threshold

### ISSUE THREAD

**Issue by magnetophon** [bill-auger/loopidity#38]
Most loopers have a very useful feature where a loop doesn't start recording until the input level crosses a certain threshold.

**Comment by bill-auger** [comment]
yes i also thought this would be handy - i had trouble at first with latency/sync/delays so i put the idea on the shelf - this feature would be tricky to get perfect tho - i dont think i would want it before the adjustable loop seams feature #14 is usable - that would at least allow to manually compensate for any over-zealous cropping
